dev/run-tests.py

```diff
@@ -687,6 +689,10 @@ def main():
         test_modules = determine_modules_to_test(changed_modules)
         excluded_tags = determine_tags_to_exclude(changed_modules)
 
+    # Run docker-integration-tests only if it's explicitly specified.
+    if not opts.modules or modules.docker_integration_tests.name not in opts.modules:
+        excluded_tags.extend(modules.docker_integration_tests.test_tags)
```

Comment:
@sarutak, this way seems okay but how about doing it with similarly with other tests like Kinesis just for consistency?  (see `git grep -r "ENABLE_KINESIS_TESTS"`).

Reply:
Thanks. I'll try it.